cmd/earthly/subcmd/billing_cmds.go

```diff
@@ -0,0 +1,128 @@
+package subcmd
+
+import (
+	"fmt"
+	"github.com/earthly/earthly/util/stringutil"
+	"strings"
+
+	"github.com/earthly/earthly/billing"
+	"github.com/earthly/earthly/cmd/earthly/helper"
+
+	"github.com/pkg/errors"
+	"github.com/urfave/cli/v2"
+)
+
+type Billing struct {
+	cli CLI
+}
+
+func NewBilling(cli CLI) *Billing {
+	return &Billing{
+		cli: cli,
+	}
+}
+
+func (a *Billing) Cmds() []*cli.Command {
+	return []*cli.Command{
+		{
+			Name:        "billing",
+			Aliases:     []string{"bill"},
+			Description: `*experimental* View Earthly billing info.`,
+			Usage:       `*experimental* View Earthly billing info`,
+			UsageText:   "earthly billing (view)",
+			Flags: []cli.Flag{
+				&cli.StringFlag{
+					Name:        "org",
```

Comment:
This still works with `earthly org select` too, right?

Reply:
yep!